spec/models/curator/shared/filestreams/file_attachments.rb

```diff
@@ -16,6 +16,7 @@
   end
 end
 
+# has_many_attached not currently used in any model, keep in case needed in the future
```

Comment:
can you add `skip => true` to the `describe` block below just to ensure we don't accidentally use it in the future if we don't want to